### ISSUE TITLE
Don't copy classes to hidden input

### ISSUE THREAD

**Issue by santyclaz** [danialfarid/ng-file-upload#651]
So I noticed in your code when creating the hidden input for doing the actual upload, it copies over every attribute from whatever element the directive is placed on.

```javascript
// angular-file-upload.js v3.2.4, lines 244-247
for (var i = 0; i < elem[0].attributes.length; i++) {
    var attribute = elem[0].attributes[i];
    fileElem.attr(attribute.name, attribute.value);
}
```

My specific issue at hand is that the additional classes messes up my styling despite the hidden input being hidden and given position absolute. I am wondering if it were possible to exclude classes from being copied over when generating this element, or whether there is a specific reason classes must also be included. Also I'm guessing it would be a good idea to exclude copying `id` attribute as well.

For those curious about the styling issue, it's because I am using bootstrap.css and it has the following CSS:

```css
.btn-block + .btn-block {
    margin-top: 5px;
}
```

So having something like this:

```html
<div classes="btn btn-block" ng-file-select ...>Upload</div>
```

results in:

```html
<input classes="btn btn-block" ... /> <!-- hidden but still causes below element to have a margin-top -->
<div classes="btn btn-block" ng-file-select ...>Upload</div>
```

**Comment by danialfarid** [comment]
gonna fix in the next version

**Comment by santyclaz** [comment]
Cool, good to know.

**Comment by danialfarid** [comment]
Fixed in version 3.3.0
